src/applications/gi/sass/partials/_gi-profile-page.scss

```diff
@@ -273,4 +273,8 @@
       display: none;
     }
   }
+
+  .sco-list {
+    list-style: none;
+  }
```

Comment:
I realize that this is the result of a refactor and not something that is being added as part of this PR, but it concerns me a bit.  This is applied to `<ul>`s that have `<li>`s of a first & last name with a title underneath, and I'm not sure that having them in a list that doesn't _look_ like a list is the best thing for accessibility.

[This article](https://unfetteredthoughts.net/2017/09/26/voiceover-and-list-style-type-none/) points out that this would cause a bug in VoiceOver:
> A bug that keeps hitting me at work is VoiceOver not announcing unordered lists properly when either the list-style-type is set to none or display is set to inline.

Setting that property changes the semantics of the HTML.  [This article](https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html) is referenced in the previous one and it discusses another solution for this problem.

tagging @1Copenut since he might have some useful insight.